TheMET/TheMET/ViewControllers/FeaturedArtViewController.swift

```diff
@@ -56,10 +57,24 @@ class FeaturedArtViewController: UIViewController {
         }
     }
     
+    @objc
+    private func favoriteServiceDidChange() {
+        guard case .loaded(let art) = self.featuredArtService.featuredArt else {
+            return
+        }
+        guard self.favoriteService.favoriteArts.contains(where: { favoriteArt in
+            return favoriteArt.objectID == art.objectID
+        }) else {
+            self.artView.isLiked = false
+            return
+        }
+        self.artView.isLiked = true
```

Comment:
So now `self.artView.isLiked` is checked and set every time the `favoriteService.favoriteArts` changes. There is no need to execute `self.artView.isLiked.toggle()` in `likeButtonDidTap` method.

Reply:
+